siocli/lib/Siocli.cpp

```diff
@@ -249,21 +249,32 @@ int commandPrompt(const ClientParameters& params)
                 command = params.m_command.get();
             } else {
                 std::ostringstream text;
-                std::size_t textLength = 0;
                 char textLastChar = '\0';
 
                 // Read command text, possibly multiline.
                 // Multiline command-text must end with a semicolon.
                 std::size_t lineNo = 0;
                 const char* prompt = kFirstLinePrompt;
+                std::size_t lineStartsInStringValue = 0;
+                std::size_t lineEndsInStringValue = 0;
+                std::size_t isInStringValue = 0;
                 do {
+                    // Prompt style
+                    if (params.m_stdinIsTerminal && lineNo == 0) std::cout << '\n';
```

Comment:
`std::cout << std::endl;` because when you showing prompt you want to flush stdout